src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("Machine started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Machines stopped working.");

```

Comment:
Similarly, the message in the 'stopWork' method should be more informative and indicate that the 'Excavator' has stopped working. For example: 'Excavator stopped working.'